diff/aclGroup.go

```diff
@@ -78,8 +79,9 @@ func (sc *Syncer) createUpdateACLGroup(aclGroup *state.ACLGroup) (*Event, error)
 		}, nil
 	}
 	if err != nil {
-		return nil, errors.Wrapf(err, "error looking up acl %v",
-			*aclGroup.Group)
+		return nil, fmt.Errorf("error looking up acl %v: %w",
```

Comment:
```suggestion
		return nil, fmt.Errorf("error looking up acl %q: %w",
```

for consistency with line 50: that one was originally quoted, this one was not